SU-SWS/decanter
# prep for drupalcamp session; general clean up and org

# READY FOR REVIEW

# Summary
- Preparation for Drupal Camp 2018 Decanter presentation
- Built out Button Element (in Core)
- Fixed issues with Icon Grid component
- Moved Icon Grid and Alerts to correct location within the Patter Library
- General clean up to comments throughout project for consistency and preparation of building out remaining items

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the Button component now exists
3. Verify that the Icon Grid component is now working/flowing mo betta
4. Comments in code are consistent (// for comments; /// for rollup and index comments)

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- Decanter

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
